### DISCUSSION TITLE
Add translation language

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [apertium/organisation#38] [General]
|language|ISO 639-3|
|--------------|-------------|
[Lojban](https://mw.lojban.org/index.php?title=Lojban&setlang=en-US)   |[jbo](https://en.wikipedia.org/wiki/Lojban)|
[Elefen](https://elefen.org/) |[lfn](https://en.wikipedia.org/wiki/Lingua_Franca_Nova)|
[Ido](https://en.wikipedia.org/wiki/Ido) |[ido](https://en.wikipedia.org/wiki/Ido)|
[Toaq](https://toaq.net/) |[qtq](https://www.kreativekorp.com/clcr/)|
[eberban](https://github.com/eberban/eberban) ||
[maj](https://github.com/sage-code/maj) ||

**Comment by mr-martian** [comment]
If you want these languages added to the web interface, you can make a PR similar to this one: https://github.com/apertium/apertium-html-tools/pull/459

Adding them as languages you can translate between is more involved. We typically name language modules based on ISO 639-3 codes, for which the following exist:
- Lojban `jbo`
- Ido `ido`
- Toaq `qtq` ([unofficial](https://www.kreativekorp.com/clcr/))

I don't see anything for the other 3, so you'd probably have to come up with private-use codes for them as well.

Once you have a language code, you can use [apertium-init](https://github.com/apertium/apertium-init) to generate the relevant data files. Since these are all conlangs, I expect they have relatively straightforward digital dictionaries which will be pretty easy to convert into the right format.

When that's done and you're ready for the actual translation part or if you need help with the dictionary conversion part let me know and I can attempt to give more specific guidance.
